shared/common-adapters/box.desktop.js

```diff
@@ -5,7 +5,7 @@ import type {Box2Props} from './box'
 
 class Box extends React.PureComponent<any> {
   render() {
-    const {forwardedRef, ...rest} = this.props
+    const {forwardedRef, inline, ...rest} = this.props
```

Comment:
why is this here?

Reply:
whoops. I did that as a temp hack. will fix

Reply:
I added inline prop to button box, and styled by default forwards all props so I was getting an error in react because div was getting a boolean.